pkg/opssight/perceivers.go

```diff
@@ -63,6 +63,36 @@ func (p *SpecConfig) ImagePerceiverReplicationController() (*components.Replicat
 	return rc, nil
 }
 
+// ArtifactoryPerceiverReplicationController creates a replication controller for the artifactory perceiver
+func (p *SpecConfig) ArtifactoryPerceiverReplicationController() (*components.ReplicationController, error) {
+	name := fmt.Sprintf("artifactory-perceiver")
+	image := fmt.Sprintf("artifactory-perceiver")
+
+	rc := p.perceiverReplicationController(name, 1)
+
+	pod, err := p.perceiverPod(name, image, util.GetResourceName(p.opssight.Name, util.OpsSightName, p.names["perceiver-service-account"]))
+	if err != nil {
+		return nil, errors.Annotate(err, "failed to create artifactory perceiver pod")
+	}
+	rc.AddPod(pod)
+	return rc, nil
+}
+
+// QuayPerceiverReplicationController creates a replication controller for the quay perceiver
+func (p *SpecConfig) QuayPerceiverReplicationController() (*components.ReplicationController, error) {
+	name := fmt.Sprintf("quay-perceiver")
+	image := fmt.Sprintf("quay-perceiver")
+
+	rc := p.perceiverReplicationController(name, 1)
+
+	pod, err := p.perceiverPod(name, image, util.GetResourceName(p.opssight.Name, util.OpsSightName, p.names["perceiver-service-account"]))
```

Comment:
perceiver service account will have god permission? do we still need them?

Reply:
I don't think we do but I thought GetResourceName just gets the name from upstream. I can create my own replication controller/ pod is that what you're saying me to do?